### PR TITLE
Add com.belmoussaoui.snowglobe.json

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*

The `--own-name=org.qemu`  is needed as the application can also run qemu itself instead of just listening on qemu running outside of the flatpak

- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
bot, build com.belmoussaoui.snowglobe

File: com.belmoussaoui.snowglobe.json [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    "command": "snowglobe",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--device=all",
+        "--socket=wayland",
+        "--own-name=org.qemu",
+        "--filesystem=home",
+        "--env=G_MESSAGES_DEBUG=all"
```
Comment (hfiguiere) [inline com.belmoussaoui.snowglobe.json:15]: why this verbosity at runtime? It will just flood the logs if anything.

Comment (bilelmoussaoui) [reply com.belmoussaoui.snowglobe.json]: Copy paste failure, thanks

**Comment by bilelmoussaoui** [comment]
bot, build com.belmoussaoui.snowglobe

**Pull request closed without merging**